README.md

```diff
@@ -41,47 +46,80 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
+* `xml.format.joinCDATALines` : Set to `true` to join lines in CDATA content during formatting. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincdatalines) for more information.
+* `xml.format.joinCommentLines` : Set to `true` to join lines in comments during formatting. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincommentlines) for more information.
+* `xml.format.joinContentLines` : Normalize the whitespace of content inside an element. Newlines and excess whitespace are removed. Default is `false`. See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincontentlines) for more information.
+* `xml.format.preserveAttributeLineBreaks`: Preserve line breaks that appear before and after attributes. This setting is overridden if `xml.format.splitAttributes` is set to `true`. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreserveattributelinebreaks) for more information.
+* `xml.format.preserveEmptyContent`: Preserve empty content/whitespace in a tag. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreserveemptycontent) for more information.
+* `xml.format.preservedNewLines`: Preserve new lines that separate tags. The value represents the maximum number of new lines per section. A value of 0 removes all new lines. Default is `2`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreservednewlines) for more information.
+* `xml.format.spaceBeforeEmptyCloseTag`: Insert space before end of self closing tag. \nExample:\n  ```<tag/> -> <tag />```. Default is `true`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatspacebeforeemptyclosetag) for more information.
+* `xml.format.splitAttributes` : Split multiple attributes each onto a new line. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatsplitattributes) for more information.
 * `xml.preferences.quoteStyle`: Preferred quote style to use for completion: `single` quotes, `double` quotes. Default is `double`.
-* `xml.autoCloseTags.enabled` : Enable/disable autoclosing of XML tags. Default is `true`. \n\nIMPORTANT: Turn off `#editor.autoClosingTags#` for this to work.
+
+* `xml.autoCloseTags.enabled` : Enable/disable autoclosing of XML tags. Default is `true`.
+  **IMPORTANT**: Turn off `#editor.autoClosingTags#` for this to work.
   **Note**: `editor.autoClosingBrackets` must be turned off to work.
 * `xml.codeLens.enabled`: Enable/disable XML CodeLens. Default is `false`.
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Codelens.md) for more information.
```

Comment:
This link is broken. It should be: `CodeLens.md`.

Reply:
fixed